### PR TITLE
add note to readme

### PR DESCRIPTION
**Pull request by hughsimpson**
This information is available in the issues, but it took me a while to figure out what was going wrong, so thought it would be good to add to the (otherwise excellent) readme

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -458,6 +458,14 @@ Make sure that `SONATYPE_PASSWORD` uses proper escaping if it contains special
 characters as documented on
 [Travis Environment Variables](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings).
 
+If your sonatype account is new (created after Feb 2021), then the default server
+location inherited from the the `sbt-sonatype` plugin will not work, and you should add
+```
+sonatypeCredentialHost := "s01.oss.sonatype.org"
+sonatypeRepository := "https://s01.oss.sonatype.org/service/local"
+```
+to your root project
```
Comment (olafurpg) [inline readme.md:467]: Should this be scoped to the root project or `ThisBuild`?

**Review by olafurpg — COMMENTED** [review]
Thank you for this contribution! I had assumed this was already documented in the readme but it's not. Would it make sense to mention this higher up in the document?

**Comment by hughsimpson** [comment]
I've moved this a bit higher up in the readme, and tried to imply that the settings should be included with the rest of the publishing settings

File: readme.md (hunk 1 above)
Comment (hughsimpson) [reply readme.md]: I... honestly don't know what it 'should' be. Best practice with `ThisBuild` in sbt has always eluded me. I think it's probably sanest to include it with the other publishing settings, so I'll try to suggest that in the copy...

**Review by olafurpg — APPROVED** [review]
Thank you!

**Pull request merged**